features/messages/src/main/kotlin/io/element/android/features/messages/timeline/components/html/DocumentProvider.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.features.messages.timeline.components.html
+
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+
+open class DocumentProvider : PreviewParameterProvider<Document> {
+    override val values: Sequence<Document>
+        get() = sequenceOf(
+            "hello",
+            // TODO Find a way to make is work with real HTML data. For now there is an error with
+            // jsoup: java.lang.NoSuchMethodError: 'java.lang.String org.jsoup.nodes.Element.normalName()'
```

Comment:
If you use `tagName` instead, the previews seem to work just fine.

Reply:
good to know, thanks. I will change and add more tests!

Reply:
`normalName` normalizes the name so always provides the lower case version, it's not the case for `tagName` which will depend on some parsing settings

Reply:
Yes, that's handled in the workaround.

The problem is `layoutlib` (used for UI preview and screenshot testing) bundles its own version of Jsoup (1.6.3) and when we try to use it for UI previews in the IDE it uses that outdated version instead of ours, where `normalName()` doesn't exist.